website/content/docs/getting-started.md

```diff
@@ -175,6 +196,59 @@ The included example topologies:
 * `TaskHookTopology.java` --- A topology that uses a task hook to subscribe to
    event notifications.
 
+## Frequently Asked Questions
+
+### 1. How do I know if my topology has been successfully submitted?
```

Comment:
s/my/a

here and in the section names below. Best to speak generally about topologies and systems without 'my' and 'your' ownership.